.eslintrc.js

```diff
@@ -34,16 +33,23 @@ module.exports = {
 		"node/no-unsupported-features": "error",
 		"node/no-deprecated-api": "error",
 		"node/no-missing-import": "error",
-		"node/no-missing-require": [
-			"error",
-			{
-				"allowModules": [
-					"webpack"
-				]
-			}
-		],
+		"node/no-missing-require": ["error", { allowModules: ["webpack"] }],
 		"node/no-unpublished-bin": "error",
 		"node/no-unpublished-require": "error",
 		"node/process-exit-as-throw": "error"
-	}
+	},
+	overrides: [
+		{
+			files: ["lib/**/*.runtime.js", "buildin/*.js", "hot/*.js"],
+			env: {
+				es6: false
+			},
+			globals: {
+				Promise: false,
```

Comment:
`env.es6=false` denies `Map`, `Set` and `Promise`.

Reply:
:+1: